core/server/common/src/test/java/alluxio/master/journal/JournalEntryAssociationTest.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.master.journal;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+import alluxio.proto.journal.Block.BlockContainerIdGeneratorEntry;
+import alluxio.proto.journal.Block.BlockInfoEntry;
+import alluxio.proto.journal.Block.DeleteBlockEntry;
+import alluxio.proto.journal.File.AddMountPointEntry;
+import alluxio.proto.journal.File.AsyncPersistRequestEntry;
+import alluxio.proto.journal.File.CompleteFileEntry;
+import alluxio.proto.journal.File.DeleteFileEntry;
+import alluxio.proto.journal.File.DeleteMountPointEntry;
+import alluxio.proto.journal.File.InodeDirectoryEntry;
+import alluxio.proto.journal.File.InodeDirectoryIdGeneratorEntry;
+import alluxio.proto.journal.File.InodeFileEntry;
+import alluxio.proto.journal.File.InodeLastModificationTimeEntry;
+import alluxio.proto.journal.File.PersistDirectoryEntry;
+import alluxio.proto.journal.File.ReinitializeFileEntry;
+import alluxio.proto.journal.File.RenameEntry;
+import alluxio.proto.journal.File.SetAclEntry;
+import alluxio.proto.journal.File.SetAttributeEntry;
+import alluxio.proto.journal.File.UpdateUfsModeEntry;
+import alluxio.proto.journal.Journal.JournalEntry;
+import alluxio.proto.journal.KeyValue.CompletePartitionEntry;
+import alluxio.proto.journal.KeyValue.CompleteStoreEntry;
+import alluxio.proto.journal.KeyValue.CreateStoreEntry;
+import alluxio.proto.journal.KeyValue.DeleteStoreEntry;
+import alluxio.proto.journal.KeyValue.MergeStoreEntry;
+import alluxio.proto.journal.KeyValue.RenameStoreEntry;
+import alluxio.proto.journal.Lineage.DeleteLineageEntry;
+import alluxio.proto.journal.Lineage.LineageEntry;
+import alluxio.proto.journal.Lineage.LineageIdGeneratorEntry;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Unit tests for {@link JournalEntryAssociation}.
+ */
+public class JournalEntryAssociationTest {
+
+  @Rule
+  public ExpectedException mThrown = ExpectedException.none();
+
+  // CHECKSTYLE.OFF: LineLengthExceed
+  private static List<JournalEntry> ENTRIES = Arrays.asList(
```

Comment:
NIT: add a comment here that this needs to be updated for every new journal entry?

Reply:
done